src/imitation/scripts/expert_demos.py

```diff
@@ -121,6 +121,7 @@ def rollouts_and_policy(
 
         if init_tensorboard:
             sb_tensorboard_dir = osp.join(log_dir, "sb_tb")
+            init_rl_kwargs = dict(init_rl_kwargs)  # Cast sacred's ReadOnlyDict to dict.
```

Comment:
Is this necessary? Are we parallelizing things inside `expert_demos`?

Reply:
Need this in order to modify `tensorboard_log` in the next line.

Reply:
Modifying the comment to be explicit why we are ~"casting" to dict here.